compat/src/index.d.ts

```diff
@@ -126,3 +126,11 @@ declare namespace React {
 		toArray: (children: preact.ComponentChildren) => preact.VNode<{}>[];
 	};
 }
+
+declare module 'preact/src/jsx' {
```

Comment:
I couldn't extend name space JSX directly above so I did this do extend it.

Reply:
I think this could be done directly in `src/jsx.d.ts` https://github.com/preactjs/preact/blob/bd3c6559fcc8a0c07798023fd9927c9bb03a844d/src/jsx.d.ts#L653

It's a global preact issue, not specifically a preact/compat one

Reply:
But does the preact without compact support ```default``` key?